angular/projects/spark-angular/src/lib/components/inputs/sprk-helpertext.stories.ts

```diff
@@ -79,9 +79,9 @@ export const helperText = () => ({
         #textInput="ngModel"
         idString="text-input-1"
       >
-      <p sprkHelperText id="helper-text">
+      <div sprkHelperText id="helper-text">
```

Comment:
Is a `<div>` tag for this helper text more semantic than using a `<p>` tag?

Reply:
I was trying to avoid a breaking change in React, but yes p tags are definitely better than divs -- I'll make the change and add it to the release notes when this is merged!